### ISSUE TITLE
Gmail rejecting message from alias

### ISSUE THREAD

**Issue by gustavokcouto** [tomav/docker-mailserver#559]
Hello,

After some days of test I finaly realized why the messages sent from my gmail to the email on the server didn't come back to gmail when i used the gmail account as an alias. I first thought that the server were not redirecting the messages, but after reading the logs in /var/log/mail/mail.log I saw that the messages were being sent back to gmail, than I changed the gmail filters not to block any messages, e I finnaly could see the messages coming,

Is there anyway to solve that? may be gmail refused the messages as they did come assigned by a gmail without being sent from a gmail account,

Thanks,
Gustavo

**Comment by tomav** [comment]
Hi @gustavokcouto, I'm sure I understand exactly what you're trying to do.
Could you give me some context and logs / config? Thanks.

**Comment by foxylion** [comment]
It may be related to my problem. When people are sending mails to an alias defined in `docker-mailserver` they are handled as spam in Google Mail.
e.g.  I have an alias rule `mailinglist@example.com -> fred@gmail.com,clara@gmail.com`
So Fred and Clara will get these mails in their spam folder.

One thing to mention here, all those mails will mostly have a failed SPF record verify, because `docker-mailserver` does no [SRS](https://en.wikipedia.org/wiki/Sender_Rewriting_Scheme). I don't know if this is enough to bring Google Mail into a position to mark the mails as spam.

Sending personal mails from `docker-mailserver` to other recipients works fine.

**Comment by Toshiwoz** [comment]
I have a similar problem, but it seems that it's my server config:
![image](https://user-images.githubusercontent.com/21297356/28879056-b131fc54-7766-11e7-8cfa-e7187053e3ea.png)

The message in the image says something like:

> Message not delivered
> --
> Impossible to deliver the message to info@my.domain because the remote server was not correctly configured. See technical details below for more information.

La risposta del server remoto è: 554 5.7.1 <info@my.domain>: Relay access denied
--
Final-Recipient: rfc822; info@my.domain
Action: failed
Status: 5.7.1
Remote-MTA: dns; my.domain (163.172.152.51, the server for the domain.)
Diagnostic-Code: smtp; 554 5.7.1 <info@my.domain>: Relay access denied
Last-Attempt-Date: Wed, 02 Aug 2017 07:37:03 -0700 (PDT)


I initially had a normal email account, no aliases.
Then I removed the account and created the alias, sending from the same server, another account and domain, it works, but if I send to that alias account from outside, like from gmail it doesn't work.

I am now trying having the actual account, but with alias too, but I think it won't work.
Suggestions?

**Comment by Toshiwoz** [comment]
I was able to solve the problem mentioned above, I created the account (i.e. info@my.domain), then the aliases file (postfix-virtual.cf) I have something like:
`info@my.domain info@my.domain,forward1@gmail.account,forward2@gmail.account`

Being:

- info@my.domain the mailserver account
- forward1@gmail.account and forward2@gmail.account the 2 gmail emails accounts where i forward the messages to

I have to verify with the second gmail account if she received the email as I can't access that. But I'm confident it worked.
